cli/aws_ddk/commands/init.py

```diff
@@ -49,12 +55,29 @@ def init_project(name: str, environment: str, template: Optional[str]) -> None:
         },
     )
 
-    # Create virtual environment (.venv)
-    echo(f"Creating virtual environment in `{path}`...")
-    cmd: str = f"{python_exec} -m venv '{path}/.venv'"
-    try:
-        run(cmd)
-    except Exception:
-        secho(f"Failed to run `{cmd}`", blink=True, bold=True)
+    if not generate_only:
```

Comment:
At the moment, if `generate_only` is `True`, the user would just see `Initializing AWS DDK project...` and then nothing would happen.

How about we add an `else` and echo that there is nothing to create because `generate_only` is `False`?

Or simply push the `echo("Done")` on line 83 out of the if statement so it always runs?

Reply:
I've pushed `echo("Done")` out of the if statement like you suggested.